src/main/scala/ch/openolitor/buchhaltung/BuchhaltungAktionenService.scala

```diff
@@ -25,38 +25,55 @@ package ch.openolitor.buchhaltung
 import ch.openolitor.core._
 import ch.openolitor.core.db._
 import ch.openolitor.core.domain._
-import ch.openolitor.core.models._
 import ch.openolitor.buchhaltung.models._
+import ch.openolitor.stammdaten.models.Person
+import ch.openolitor.core.models.PersonId
 import scalikejdbc._
 import com.typesafe.scalalogging.LazyLogging
-import akka.actor.ActorSystem
-import ch.openolitor.core.Macros._
+import akka.actor.{ ActorRef, ActorSystem }
+import akka.pattern.ask
+import akka.util.Timeout
+import ch.openolitor.core.mailservice.MailService._
+import ch.openolitor.mailtemplates.engine.MailTemplateService
 import ch.openolitor.core.Macros._
 import ch.openolitor.buchhaltung.BuchhaltungCommandHandler._
+import ch.openolitor.buchhaltung.eventsourcing.BuchhaltungEventStoreSerializer
 import ch.openolitor.buchhaltung.models.RechnungModifyBezahlt
 import ch.openolitor.buchhaltung.repositories.DefaultBuchhaltungWriteRepositoryComponent
 import ch.openolitor.buchhaltung.repositories.BuchhaltungWriteRepositoryComponent
 import ch.openolitor.core.repositories.EventPublishingImplicits._
 import ch.openolitor.core.repositories.EventPublisher
 
+import scala.util.{ Failure, Success }
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext.Implicits.global
+
 object BuchhaltungAktionenService {
-  def apply(implicit sysConfig: SystemConfig, system: ActorSystem): BuchhaltungAktionenService = new DefaultBuchhaltungAktionenService(sysConfig, system)
+  def apply(implicit sysConfig: SystemConfig, system: ActorSystem, mailService: ActorRef): BuchhaltungAktionenService = new DefaultBuchhaltungAktionenService(sysConfig, system, mailService)
 }
 
-class DefaultBuchhaltungAktionenService(sysConfig: SystemConfig, override val system: ActorSystem)
-  extends BuchhaltungAktionenService(sysConfig) with DefaultBuchhaltungWriteRepositoryComponent {
+class DefaultBuchhaltungAktionenService(sysConfig: SystemConfig, override val system: ActorSystem, override val mailService: ActorRef)
+  extends BuchhaltungAktionenService(sysConfig, mailService) with DefaultBuchhaltungWriteRepositoryComponent {
 }
 
 /**
  * Actor zum Verarbeiten der Aktionen für das Buchhaltung Modul
  */
-class BuchhaltungAktionenService(override val sysConfig: SystemConfig) extends EventService[PersistentEvent] with LazyLogging with AsyncConnectionPoolContextAware
-  with BuchhaltungDBMappings {
+class BuchhaltungAktionenService(override val sysConfig: SystemConfig, override val mailService: ActorRef) extends EventService[PersistentEvent]
+  with LazyLogging
+  with AsyncConnectionPoolContextAware
+  with BuchhaltungDBMappings
+  with MailServiceReference
+  with BuchhaltungEventStoreSerializer
+  with MailTemplateService
+  with SystemConfigReference {
   self: BuchhaltungWriteRepositoryComponent =>
 
   val False = false
   val Zero = 0
 
+  implicit val timeout = Timeout(15.seconds)
```

Comment:
@AlwinEgger Do we want a global actor timeout of 15 Seconds? Or only SendEmail?

Reply:
It can be an option to increase it overall. But then it would be nicer to specify it in the config-file.

Reply:
extracted to the config file. In case this entry is not present, 15 seconds is the default